src/scss/_admonitions.scss

```diff
@@ -46,109 +52,109 @@
   }
 
   &.attention {
-    border-color: $orange;
+    border-color: rgba(var(--color-admonition-warning), 1);
     .admonition-title {
-      background-color: $orange-light;
+      background-color: rgba(var(--color-admonition-warning), 0.1);
```

Comment:
Nice! Is this the reason that the colors need to be defined as a RGB tuple?

Reply:
The RGB tuple is the required argument for the `rgba()` function.